earthpy/plot.py

```diff
@@ -123,17 +123,23 @@ def plot_bands(
         "Input arr should be a numpy array"
 
     if title:
+        if isinstance(title, str):
+            title = [title]
+
+        # A 2-dim array should only be passed one title
         if (arr.ndim == 2) and (len(title) > 1):
             raise ValueError(
                 """Plot_bands() expects one title for a single
```

Comment:
Two minor suggestions to improve these error messages: 

1. We should probably use `plot_bands` instead of `Plot_bands`, even though it is the first word in a sentence, so that the error message has the same capitalization as the function. 

2. Tim snuck a handy trick for multiline error messages in the io module that is a little cleaner and does not result in the awkward tabs that come with the use of `"""` with indendation. See https://github.com/earthlab/earthpy/blob/master/earthpy/io.py#L163 for an example, but the idea is to do something like:

```python
raise ValueError(
  "plot_bands expects..."
  "and the error message continues..."
)
```